java/vector/src/test/java/org/apache/arrow/vector/file/json/TestJSONFile.java

```diff
@@ -156,6 +161,48 @@ public void testWriteReadDateTimeJSON() throws IOException {
   }
 
   @Test
+  public void testWriteReadDictionaryJSON() throws IOException {
```

Comment:
Yeah, I definitely want to do that.  I was just a little pressed for time and wanted to put what I had so far out.